api/src/main/resources/workbench-api.yaml

```diff
@@ -6311,18 +6311,18 @@ definitions:
       duaTypeEnum:
         "$ref": "#/definitions/DuaType"
         description: 'Type of Data Use Agreement Institution has signed, Master or restricted to a defined set of specific researchers'
-      emailAddresses:
-        type: array
+      tierEmailAddresses:
         description: A list of email addresses which can be validated as members of
-          this Institution
-        items:
-          type: string
-      emailDomains:
+          this Institution for each access tier.
         type: array
+        items:
+          "$ref": "#/definitions/TierEmailAddresses"
+      tierEmailDomains:
```

Comment:
Note: this appears to be a breaking change, but it only really affects the institution admin pages AFAICT , so it's probably fine

Reply:
Yup, I mentioned that in the PR description above. Also the API usage is very low like 1 every few weeks/months, so I think it should be fine.